include/rapidcheck/detail/Unicode.hpp

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+namespace rc {
+namespace detail {
+
+template<typename T, typename RandomType>
```

Comment:
Any particular reason this needs to be templated on `T`? You could just use a type that is large enough. Also, since `RandomType` is unlikely to be anything other than `Random`, this likely doesn't need to be a template at all.